docs/lake/design.md

```diff
@@ -416,7 +442,7 @@ the special sub-pool name `<pool>:journal`.
 For example, to aggregate a count of each journal entry type of the pool
 called `logs`, you can simply say:
 ```
-zed lake query -p logs:journal "count() by typeof(.)"
+zed lake query "from logs:journal | count() by typeof(this)"
```

Comment:
I updated this to phase out the `-p` flag since we know that's coming back, but (per the note just below this example) this journal sub-pool query isn't implemented yet, so I'm basically guessing this will still be the future syntax. Also, I'd like to do my link-to-the-issue-not-yet-implemented thing in that note below, but I'm not sure if #2547 is the correct one or if we have to open a new one. @mccanne could you confirm?